drift/lib/src/query_builder/statements/query.dart

```diff
@@ -15,14 +15,12 @@ abstract mixin class SingleTableStatementMixin<
     RS extends ResultSet<Row, RS>,
     Self extends SingleTableStatementMixin<Row, RS, Self>> {
```

Comment:
Is there another place where `Self` is still used now? If not, I think we can get rid of this.